src/MenuScreen.cpp

```
@@ -115,4 +115,17 @@ MenuScreen::MenuScreen(MenuItem** items) : items(items) {
     while (items[itemCount] != nullptr) {
         itemCount++;
     }
+}
+
+void MenuScreen::poll(MenuRenderer* renderer, uint16_t pollInterval) {
```

cppcheck (2.10-2)

[style] The function 'poll' is never used.

(unusedFunction)